src/components/app/app.html

```diff
@@ -2,7 +2,11 @@
   <div class="app__drawer-point"></div>
   <div class="app__container">
     <div class="app__container-toolbar-point"></div>
-    <div class="app__container-content">
+    <div class="app__container-content mdc-layout-grid">
+      <div class="mdc-layout-grid__inner">
+        <div class="app__pie-chart mdc-card mdc-layout-grid__cell mdc-layout-grid__cell--span-6"></div>
```

Comment:
![image](https://user-images.githubusercontent.com/36405386/37306410-51e15e46-2640-11e8-97c3-66fe4d5a7267.png)

please either use flexbox instead of grid or fix this behavior on smaller screens (second column dissapears)

Reply:
you probably could use just flexbox instead of grid (but it's up to you)